01/app.js

```diff
@@ -0,0 +1,22 @@
+const formEl = document.querySelector('form')
+const firstNameInputEl = document.querySelector('input[ name="firstName"]')
+const lastNameInputEl = document.querySelector('input[ name="lastName"]')
+const userList = document.querySelector('.users-list')
+
+
+//listener
+formEl.addEventListener('submit', (e)=> {
+    e.preventDefault()
+
+    const firstName = firstNameInputEl.value
+    const lastName = lastNameInputEl.value
```

Comment:
Warto sprawdzić czy pomocy `if` czy te wartości nie są puste. Jeśli są to trzeba poprosić użytkownika przez `alert`, aby je uzupełnił :)